packages/adp-tooling/src/types.ts

```diff
@@ -35,21 +35,41 @@ export interface AdpWriterConfig {
     target: AbapTarget;
     ui5?: {
         minVersion?: string;
+        version?: string;
+        url?: string;
     };
     package?: {
         name?: string;
         description?: string;
     };
+    flp?: {
```

Comment:
That object is odd. `bspName` and `credentials` are used to access the original app while `title`, `subtitle` and `inboundId` are only being used in the i18n file. Did you miss adding a change to the appdescr if these properties are present or did I miss it in the code or is there some magic backend functionality that does something on deploy? 

What is the difference between `bspName` and `reference` in the app section?

Reply:
Currently, the title, subtitle and input id are only used in the i18n file because the appdescr information comes already filtered after the prompting.
bspName is the base app repoName  and reference is the base app id

Reply:
As discussed on the phone, please add the change for the inbound communication to the appdescr template file with the condtion that the flp object is provided

Reply:
The change for the inbound it's more complex to be in the aprdescr template and because of that it's passed with other changes in the content

Reply:
If it is too complex for ejs, implement it as a plain function and then add it to the list of changes. Most importantly move the logic here because you have a reference to it in your generated i18n.properties file